go/logger/standard.go

```diff
@@ -101,6 +101,7 @@ func (log *Standard) initLogging() {
 	// to point to the appropriate log file.
 	initLoggingBackendOnce.Do(func() {
 		logBackend := logging.NewLogBackend(os.Stderr, "", 0)
+		logBackend.Color = true
```

Comment:
This is really complicated but it is still needed on Windows, and it does not break logging. I will test on linux first, but I think colors worked on 'nix without this because of doFmtVerbLevelColor, which is a no-op on Windows. The reason colors don't go into the logging file is because of logger.fileFormat.